rewrite-test/src/main/java/org/openrewrite/test/SourceSpec.java

```diff
@@ -65,6 +65,11 @@ public SourceSpec<T> path(Path sourcePath) {
         return this;
     }
 
+    public SourceSpec<T> path(String sourcePath) {
```

Comment:
I'm glad you picked this up - I didn't know about the `SourceSpec` and `SourceSpecs`, I'll look at this Monday.